src/site/content/en/learn/forms/form-fields/index.md

```diff
@@ -97,7 +97,7 @@ different validation rules, and more. Let's see how to change the type.
 
 By using `type="checkbox"` the browser now renders a checkbox instead of a text field.
 The checkbox also comes with additional attributes.
-You can set the `selected` attribute, to show it as selected.
+You can set the `checked` attribute, to show it as selected.
```

Comment:
```suggestion
You can set the `checked` attribute, to show it as checked.
```

Reply:
Totally minor suggestion, but "selected" kind of makes me think of a select box (and I believe that's the attribute for a select box).

Reply:
yeah i feel also the same. Will update !